csharp/src/Ice/Internal/Transceiver.cs

```diff
@@ -44,8 +44,8 @@ public interface Transceiver
     /// <param name="buf">The buffer to write.</param>
     /// <param name="callback">The callback that this operation calls when it completes asynchronously.</param>
     /// <param name="state">The state to pass to the callback.</param>
-    /// <param name="messageFullyWritten">When <see langword="true" />, the operation has written or is in the process of
-    /// writing the remainder of the message; otherwise, <see langword="false"/>.</param>
+    /// <param name="messageFullyWritten">When <see langword="true" />, the operation has written or is in the process
```

Comment:
Given the revised logic, we should rename messageFullyWritten => messageWritten. For a follow-up PR.